content/docs/0.8.x/monitoring/prometheus/install/index.md

```diff
@@ -82,7 +101,33 @@ keptn configure monitoring prometheus --project=sockshop --service=carts
 * To verify that the Prometheus scrape jobs are correctly set up, you can access Prometheus by enabling port-forwarding for the prometheus-service:
 
 ```BASH
-kubectl port-forward svc/prometheus-service 8080 -n <PROMETHEUS_NS>
+kubectl port-forward svc/prometheus-service 8080 -n default
```

Comment:
Please change `-n default` to `-n keptn` since you installed it in the keptn namespace above.

Reply:
@johannes-b , We are installation Prometheus in default at beginning of this doc. so it should in default

```bash
helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
helm install prometheus prometheus-community/prometheus --namespace default
```